migration-engine/connectors/sql-migration-connector/src/sql_schema_calculator.rs

```diff
@@ -119,13 +119,14 @@ impl<'a> SqlSchemaCalculator<'a> {
                 })
                 .collect();
 
-            let primary_key = sql::PrimaryKey {
+            let primary_key = Some(sql::PrimaryKey {
                 columns: model
                     .id_fields()
                     .map(|field| field.db_name().to_owned())
                     .collect(),
                 sequence: None,
-            };
+                constraint_name: None,
+            }).filter(|pk| !pk.columns.is_empty());
```

Comment:
It feels like there's a lot going on this block -- a negated search, mapping id fields to database names, initializing variables. And then finally wrapping it all into a `Some`. It might be worth taking the opportunity to untangle these slightly.

Reply:
good point, I'll check if I can make this one more readable.